index.html

```diff
@@ -13,14 +13,17 @@
             <div class="name">
                 <h1>Name</h1>
             </div>
-            <div>
-                <ul class="nav-list">
-                    <li><a href="#">Home</a></li>
-                    <li><a href="#">Skills</a></li>
-                    <li><a href="#">Education</a></li>
-                    <li><a href="#">Experience</a></li>
-                    <li><a href="#">Contact</a></li>
-                </ul>
+            <ul class="nav-list">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Skills</a></li>
+                <li><a href="#">Education</a></li>
+                <li><a href="#">Experience</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
```

Comment:
Change the order to Home Education Skills Experience Contact

Reply:
Done

Reply:
Change the order in which they are shown.
I don't think it is done yet.
<li><a href="#">Home</a></li>
                <li><a href="#">Education</a></li>
                <li><a href="#">Skills</a></li>
                <li><a href="#">Experience</a></li>
                <li><a href="#">Achievements</a></li>
                <li><a href="#">Contact</a></li>